portfolio/src/main/webapp/indexScript.js

```diff
@@ -0,0 +1,53 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+  getBlogs();
+  const selectElement = document.getElementById('cat-select');     
+  selectElement.addEventListener('change', function() {
+    getBlogs();
+  });
+});
+
+function getBlogs() {
+  fetch('/post').then(response => response.json()).then((posts) => {
+    const blogListElement = document.getElementById('blog-posts');     
+    const selectedElement = document.getElementById('cat-select');
+    blogListElement.innerHTML = '';  
+    for (post of posts) {
+      if ((selectedElement.value === 'All') || (post.category === selectedElement.value)) {
+          blogListElement.appendChild(createBlogElement(post));
+      }
+    }
+  });
+}
+
+function createBlogElement(post) {
+  const cardElement = document.createElement('div');
+  cardElement.className = 'blog-card';
+
+  const imgElement = document.createElement('img');
+  imgElement.src = 'images/ai.jpg';
+  imgElement.width = '300';
+  imgElement.height = '200';
+
+  const contentElement = document.createElement('div');
+  contentElement.className = 'content';
+
+  const titleElement = document.createElement('h3');
+  titleElement.innerText = post.title;
+
+  const blurbElement = document.createElement('p');
+  blurbElement.innerText = (post.blogpost).length < 25 ? post.blogpost : post.blogpost.substring(0,200);
```

Comment:
extra parens

Reply:
Although I'm pretty confused by this, why do you need to check if less than 25?

Reply:
I'm trying to retrieve the first 25 characters of the blog post to display in the preview of the blog cards, but if there aren't over 25 characters, it should just display the entire blog post. At least that's what I was trying to go for.

Reply:
Oh wait, I definitely should be making this 200 instead of 25